source_test.yaml

```
@@ -209,7 +209,7 @@
   type: 0
   ignore_patterns: ['^(?!OSV-).*$']
   directory_path: 'vulns'
-  repo_url: 'ssh://github.com/google/oss-fuzz-vulns'
+  repo_url: 'https://github.com/google/oss-fuzz-vulns'
```

nit: this is technically more correct
```suggestion
  repo_url: 'https://github.com/google/oss-fuzz-vulns.git'
```